x-pack/plugins/index_management/__jest__/client_integration/index_details_page/select_inference_id.test.tsx

```diff
@@ -23,7 +23,13 @@ const mockDispatch = jest.fn();
 jest.mock('../../../public/application/app_context', () => ({
   useAppContext: jest.fn().mockReturnValue({
     core: { application: {} },
-    docLinks: {},
+    docLinks: {
+      links: {
+        enterpriseSearch: {
+          inferenceApiCreate: 'https://abc.com/inference-api-create',
```

Comment:
@saikatsarkar056 this is a placeholder url

Reply:
It's in a test so that's fine, though it's better to  use a URL that doesn't resolve to a real existing website.